lib/src/events.dart

```diff
@@ -13,8 +13,17 @@ class Emitter {
 
   Stream<Event> operator[] (type) => _evtController.stream.where((e) => e.type == type);
 
+  Stream<T> getEventStream<T extends Event>(String type) {
+    final controller = new StreamController<T>();
+    _evtController.stream.where((e) => e.type == type).listen(
+        controller.add,
```

Comment:
Kind of nitpicky, but I think the `StreamController` will continue to get events after the subscription ends, right? 

The source stream would then have to remain active continuing to emit events even when no one is consuming them, and you'd have to use the `onListen`/`onCancel` on the controller constructor to manage that. Not sure what the behavior of pausing/resuming would be, either.

I think you could use `.map` until `whereType` is implemented, which would be a simpler solution and cover all of the stream consumption edge cases:

```dart
Stream<T> getEventStream<T>(String type) {
  return _evtController.stream.where((e) => e.type == type).map((e) => e as T);
}
```
Could even get fancy if you don't want to incur any potential `as` overhead:
```dart
Stream<E> getEventStream<E>(String type) {
  return _evtController.stream.where((e) => e.type == type).map<T>(_implicitDowncast);
}
```
```dart
B _implicitDowncast<A, B extends A>(A a) {
  B b = a;
  return b;
}
```